File: app/src/main/java/com/example/workoutapp/Workout.java
package com.example.workoutapp;

import java.util.ArrayList;

public class Workout {
    String title;
    String content;

    public Workout() {};

    public Workout(String title, String content){
            this.title = title;
            this.content = content;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: app/src/main/java/com/example/workoutapp/Workout.kt
package com.example.workoutapp

class Workout {
    @JvmField
    var title: String? = null
    @JvmField
    var content: String? = null

    constructor()
    constructor(title: String?, content: String?) {
        this.title = title
        this.content = content
    }
}
